scripts/controllers/Lobby.gd

```diff
@@ -0,0 +1,29 @@
+extends Control
+
+
+func _ready():
+	var _junk = get_tree().connect("connected_to_server", self, "connected")
+
+func connected():
+	print("Connected!")
+	if Net.is_host and Net.current_players == Net.MAX_PLAYERS:
+		
+		## TODO: Set the random seed here, and broadcast it to all players before the game starts
+		rpc("begin_game")
+		begin_game()
+	else:
+		if not Net.is_host:
+			rpc_id(0, "player_joined")
+
+
+
+remote func player_joined():
```

Comment:
It'd be nice to have a player_left method that handled peer_disconnected and decremented the players / cleaned-up the relevant player node